test/detecturl_test.js

```diff
@@ -53,4 +53,18 @@ suite('detectURL', function() {
       assert.equal(error.message, 'Could not find appropriate artifact');
     }
   });
+
+  test('emulator-kk', async function() {
+    let options = {
+      product: 'emulator-kk',
+      os: 'linux-x86_64',
+      branch: 'mozilla-central'
+    };
+
+    try {
+      await detectURL(options);
+    } catch (error) {
+      assert.equal(error.message, 'Could not find appropriate artifact');
```

Comment:
Er we want to be able to find the emulator, right? There's a problem if we can't find it